src/test/java/bio/terra/common/auth/AuthService.java

```diff
@@ -28,34 +27,34 @@
 
 @Component
 public class AuthService {
-  private static Logger logger = LoggerFactory.getLogger(AuthService.class);
-  private static ExpirationPolicy<String, GoogleCredential> TOKEN_CACHE_EXPIRATION_POLICY =
+  private static final Logger logger = LoggerFactory.getLogger(AuthService.class);
+  private static final ExpirationPolicy<String, GoogleCredential> TOKEN_CACHE_EXPIRATION_POLICY =
       // Make sure this value never returns a negative since that means the entry never expires
       (key, value) ->
           Math.max(0, value.getExpirationTimeMilliseconds() - TimeUnit.MINUTES.toMillis(5));
 
   // the list of scopes we request from end users when they log in.
   // this should always match exactly what the UI requests, so our tests represent actual user
   // behavior:
-  private List<String> userLoginScopes =
+  private final List<String> userLoginScopes =
       List.of(
           "openid", "email", "profile", "https://www.googleapis.com/auth/cloud-billing.readonly");
-  private List<String> directAccessScopes =
+  private final List<String> directAccessScopes =
       List.of(
           "https://www.googleapis.com/auth/bigquery",
           "https://www.googleapis.com/auth/devstorage.full_control");
-  private NetHttpTransport httpTransport;
-  private JacksonFactory jsonFactory = JacksonFactory.getDefaultInstance();
+  private final NetHttpTransport httpTransport;
+  private final GsonFactory jsonFactory = GsonFactory.getDefaultInstance();
```

Comment:
Why switch between JacksonFactory and GsonFactory?

Reply:
`JacksonFactory` is deprecated and `GsonFactory` is the recommended replacement https://cloud.google.com/java/docs/reference/google-http-client/latest/com.google.api.client.json.jackson2.JacksonFactory